src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionTestResultViewController.swift

```diff
@@ -33,6 +33,10 @@ class ExposureSubmissionTestResultViewController: DynamicTableViewController, EN
 
 		setUpView()
 		setUpBindings()
+		
+		footerView?.primaryButton?.accessibilityIdentifier = AccessibilityIdentifiers.ExposureSubmission.primaryButton
+		footerView?.secondaryButton?.accessibilityIdentifier = AccessibilityIdentifiers.ExposureSubmission.secondaryButton
```

Comment:
I wonder if we cannot simply set a default accessibility ID for the footer buttons. They are unique per screen.
@nickguendling @marcussc ?

Reply:
Sounds good 👍

Reply:
Will be added later.

Reply:
Yes